package.json

```diff
@@ -46,7 +46,6 @@
     "@ls-lint/ls-lint": "^2.2.2",
     "@parcel/watcher": "^2.3.0",
     "@taplo/cli": "^0.7.0",
-    "chalk": "^4",
```

Comment:
Thanks. But we think using `chalk` is alright. It's only for devs not for users.

Reply:
> Thanks. But we think using `chalk` is alright. It's only for devs not for users.

Thank you for your reply.